src/components/ApplicationMenu/ApplicationMenu.js

```diff
@@ -35,24 +35,16 @@ class ApplicationMenu extends Component<Props> {
   menu: any;
 
   componentDidMount() {
-    setTimeout(() => {
-      this.buildMenu();
-    }, 1000);
+    this.buildMenu(this.props);
```

Comment:
@melanieseltzer because `<ApplicationMenu />` is now within the `isAppLoaded` check (in `App.js` above), we no longer need the setTimeout, since `devServerTask` is known on mount :D